bundles/org.openhab.ui/web/src/components/developer/developer-sidebar.vue

```diff
@@ -450,6 +450,9 @@ export default {
   },
   mounted () {
     this.startEventSource()
+    setTimeout(() => {
```

Comment:
This is usually only done for desktop, because on mobile this would bring up the on screen keyboard which would be very annoying.
And you could use nextTick instead of a timeout.

See https://github.com/openhab/openhab-webui/blob/77dbfcb198d20053b55373f32255f2a50eaa2ff7/bundles/org.openhab.ui/web/src/pages/settings/things/add/choose-binding.vue#L85-L89

Reply:
Thanks for the tip!